### PR TITLE
fix(icon-button-bar): use selectorPrimaryFocus in OverflowMenu

### PR DESCRIPTION
**Pull request by jendowns**
## Affected issues

partially addresses #683 

## Proposed changes

- use `selectorPrimaryFocus` in `OverflowMenu` (set it to a known class name for icon buttons)
- add `setFocus` prop to individual actions (assign it a class name if true)

## Testing instructions

- ensure that `IconButtonBar` functions as expected

### CODE REVIEW

File: src/components/IconButtonBar/IconButtonBar.js [hunk 1]
```diff
@@ -73,6 +74,7 @@ const IconButtonBar = ({
         key={action.label}
         onClick={action.onClick}
         disabled={action.disabled}
+        {...action.otherProps}
```
Comment (SimonFinney) [inline src/components/IconButtonBar/IconButtonBar.js:77]: Is there an opportunity to spread these flat from the `action` as opposed to having them contained within an `otherProps` property? I'm unfamiliar with this pattern if it's in use elsewhere

Comment (jendowns) [reply src/components/IconButtonBar/IconButtonBar.js:77]: Consider that the `selectorPrimaryFocus` targets a selector in a child `OverflowMenuItem`, which can be anything the user wants.

**Pull request merged**